webapp/src/lib/config.ts

```diff
@@ -43,6 +54,7 @@ export class Config {
         return new Config(
             getField("ByzCoinID", asID),
             Roster.fromTOML(raw),
+            tryToGetField("SignupNode", asString),
```

Comment:
consider parsing it as URL to ensure that it is in the correct format and fails at load time, not latter

Reply:
Added `asURL`, even though for the `ServerIdentity` I need to convert it back to a string...

Reply:
yep, ideally, `ServerIdentity` should take a URL, but for now, at least the configuration will be correct.

I remember that some times ago, I already wanted to change cothority to use the correct URL type, to catch misconfiguration early, but it was cumbersome as it wasn't working well with protobuf.